apps/src/lab2/ai/AiTutorManager.ts

```diff
@@ -0,0 +1,49 @@
+import {postAichatCompletionMessage} from '@cdo/apps/aichat/aichatApi';
+import {PendingChatMessage, AichatContext} from '@cdo/apps/aichat/types';
+import {EMPTY_AI_CUSTOMIZATIONS} from '@cdo/apps/aichat/views/modelCustomization/constants';
+import {Role} from '@cdo/apps/aiComponentLibrary/chatMessage/types';
+import {getStore} from '@cdo/apps/redux';
+import {
+  AiChatModelIds,
+  AiInteractionStatus as Status,
+} from '@cdo/generated-scripts/sharedConstants';
+
+const systemPrompts = {
+  hint: "You are responding to a query about programming.  Target the reading age of an American 7th grader.  Use the Socratic method to guide the student to the answer, but do not give them the answer directly.  Just focus on the biggest single issue you find.  Use plain English in the answer.  I don't want multiple steps, points, or questions.  Just one question that helps the student to make progress.",
+  user: 'You are responding to a query about programming.  Target the reading age of an American 7th grader.  Use plain English in the answer.  Keep the answer relatively short, say three or fewer paragraphs, with each paragraph three sentences or less.',
+};
+
+export default class AiTutorManager {
```

Comment:
Out of curiosity, any reason this can't just be an async function call (as opposed to a class) since it's not managing any state internally?

Reply:
The opportunity here is to instantiate multiple of these (though likely `Manager` isn't the right suffix) and then each can take care of an ongoing chat session.